charts/sonarqube-dce/templates/application.yml

```diff
@@ -36,7 +38,7 @@ spec:
     matchLabels:
       # we should use app.kubernetes.io/name, which should be set by default to all pods, but this is not working.
       # so we rely on our own custom release field
-      release: "{{ .Release.Name }}" 
+      release: "{{ .Release.Name }}"
```

Comment:
This is no longer relevant, has been superseded by https://github.com/SonarSource/helm-chart-sonarqube/commit/60f06c7b0bb1fef334ec5839e2ac885f8a344ad3.
Rebase might be needed ?